library/src/main/java/com/dslplatform/json/JsonWriter.java

```diff
@@ -153,7 +153,7 @@ public final void writeByte(final byte value) {
 	 *
 	 * @param value string to write
 	 */
-	public final void writeString(final String value) {
+	public final void writeString(final CharSequence value) {
```

Comment:
I'm wondering if there will be some perf regression due to this change. I can test it later, but unless you can provide some numbers I would be much more comfortable if you copy this method instead.